orchagent/copporch.cpp

```diff
@@ -69,7 +69,11 @@ static map<string, sai_hostif_trap_type_t> trap_id_map = {
     {"router_custom_range", SAI_HOSTIF_TRAP_TYPE_ROUTER_CUSTOM_RANGE_BASE},
     {"l3_mtu_error", SAI_HOSTIF_TRAP_TYPE_L3_MTU_ERROR},
     {"ttl_error", SAI_HOSTIF_TRAP_TYPE_TTL_ERROR},
-    {"udld", SAI_HOSTIF_TRAP_TYPE_UDLD}
+    {"udld", SAI_HOSTIF_TRAP_TYPE_UDLD},
+    {"bfd", SAI_HOSTIF_TRAP_TYPE_BFD},
+    {"bfdv6", SAI_HOSTIF_TRAP_TYPE_BFDV6},
+    {"src_nat_miss", SAI_HOSTIF_TRAP_TYPE_SNAT_MISS},
```

Comment:
Almost all above follows the suffix of SAI type, so shouldn't it be `snat_miss` instead of `src_nat_miss`. Just an observation.

Reply:
Yes. You are correct that we should stick to the common naming convention.  However, since this is done by other team, I will try to remove these two new trap from the submission if I can make the code change.  Or else I will have to abandon this submit and do a separate submission later.  I will try to abandon this one first.
Thanks,
Jingping